Yoast/yoast-components
# Fix positioning of replacement variable suggestions

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Uses and overrides the `positionSuggestions` of the Draft.js mention plugin.
* Used constant for tweaking of the offset for a slightly more pleasing visual.
* Binds to the window scroll event to force update the mention suggestions position. **Any alternatives to resolve this are welcome.**

## Test instructions

This PR can be tested by following these steps:

* Test in wordpress-seo and standalone both.
* When the suggestions are close to the right and bottom edge it should now move the popover instead of going off the screen.
* Should now update the position to the preferred spot on scrolling.

Fixes https://github.com/Yoast/wordpress-seo/issues/9604
